flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/source/split/MySqlBinlogSplit.java

```diff
@@ -108,6 +108,16 @@ public boolean isCompletedSplit() {
         return totalFinishedSplitSize == finishedSnapshotSplitInfos.size();
     }
 
+    public String getTables() {
+        String tables;
+        if (tableSchemas != null) {
+            tables = tableSchemas.keySet().toString();
```

Comment:
If we iterate every time, it will have an impact on performance. Can we assume that tableschemas will not change and we only need to build the tables variable once.

Reply:
Good advice, I will refine it.

Reply:
Resolved.